app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: post, local: true) do |form| %>
```

Comment:
Here it seem to auto generate front-end, but if yor are editting the front-end the gem slim allow you to edit RoR frontend easier.